ci/ci.sh

```diff
@@ -3,9 +3,17 @@ set -xe
 
 # Defaults (in case they are not set by the CI)
 F=${F:-production}
-DC=${DC:-dmd1}
 DIST=${DIST:-xenial}
 
+# Binary/package name deduced based on supplied DMD version
+case "$DMD" in
+    1.*    ) DC=dmd1 ;;
+    2.*.s* ) DC=dmd-transitional ;;
+    *      ) DC=dmd ;;
+esac
+
+apt install -y $DC=$DMD-$DIST
```

Comment:
For now, as a workaround, you can do the following (eventually I will integrate this into beaver, but we can give it a go first here):

```yml
install: beaver docker build --build-arg "DMD=$DMD" -f "Dockerfile${DIST:+.$DIST}" .
```

(this is basically what `beaver install` does plus passing the `DMD` argument to the build command).

Then in the `Dockerfile`:
```
ENV DMD=$DMD
```

Then you can use the `DMD` env variable from inside the build script.